srcpkgs/python3-jq/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-jq'
+pkgname=python3-jq
+version=1.2.2
+revision=1
+wrksrc="${pkgname#*-}.py-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Cython"
+makedepends="python3-devel jq-devel"
```

Comment:
`$depends` is missing

Reply:
While checking the runtime deps I noticed that the library that python3-jq links against is provided by jq-devel, not jq. Should I change jq's template so those files aren't moved into jq-devel?

Reply:
I meant that python3 is missing from $depends, but that's a good catch. cc @leahneukirchen